pkg/network/event_common.go

```diff
@@ -214,7 +214,7 @@ type ConnectionStats struct {
 	IPTranslation *IPTranslation
 	Via           *Via
 
-	Monotonic StatCounters
+	Monotonic map[uint32]StatCounters
```

Comment:
maps can be large in practice. Have you examined the effect this has on memory usage?

Reply:
3% increase in memory

![system-probe RSS (avg) (4)](https://user-images.githubusercontent.com/6599778/181628436-3de36885-766c-4f32-add0-2fc4cf16995b.png)